llvm/include/llvm/IR/IntrinsicsWebAssembly.td

```diff
@@ -321,6 +321,18 @@ def int_wasm_relaxed_dot_bf16x8_add_f32:
                         [llvm_v8i16_ty, llvm_v8i16_ty, llvm_v4f32_ty],
                         [IntrNoMem, IntrSpeculatable]>;
 
+//===----------------------------------------------------------------------===//
+// Half-precision intrinsics (experimental)
+//===----------------------------------------------------------------------===//
+
+// TODO: Replace these intrinsic with normal ISel patterns once the XXX
+// instructions are merged to the proposal.
+def int_wasm_loadf16_f32:
+  Intrinsic<[llvm_float_ty],
+            [llvm_ptr_ty],
+            [IntrReadMem, IntrArgMemOnly],
+             "", [SDNPMemOperand]>;
```

Comment:
Why is the name empty?

Reply:
It looks like we have empty/missing names for nearly all of the intrinsics except for `int_wasm_ref_is_null_extern` and `int_wasm_ref_is_null_func`. Do you know what the name is used for? Maybe it's for when the name can't be automatically translated to an llvm name?

Reply:
Hmm I'm not sure. If this works without the name I guess it's fine.